src/Servers/Kestrel/tools/CodeGenerator/TransportMultiplexedConnectionFeatureCollection.cs

```diff
@@ -16,13 +16,15 @@ public static string GenerateFile()
                 "IConnectionIdFeature",
                 "IConnectionTransportFeature",
                 "IConnectionItemsFeature",
+                "IPersistentStateFeature",
```

Comment:
Do we plan to pool any TransportMultiplexedConnections? I think we should just add this later if needed.

Reply:
I still don't see why TransportMultiplexedConnection needs to change at all. These are never pooled, right?

Reply:
Was looking at outdated commit.